eth/vm/base.py

```diff
@@ -582,7 +582,7 @@ def validate_header(self,
                     self.validate_seal(header)
                 except ValidationError:
                     self.cls_logger.warning(
-                        "Failed to validate header proof of work on header: %r",
+                        "Failed to validate seal on header: %r",
```

Comment:
Would it make sense to include the exception msg here as well as it might include more info about why exactly it failed?

Reply:
Oh yeah good idea, I'll add that :+1: